src/librbd/internal.cc

```diff
@@ -318,7 +318,6 @@ int mirror_image_disable_internal(ImageCtx *ictx, bool force) {
         " disable mirroring" << dendl;
       return -EINVAL;
     }
-    goto remove_mirroring_image;
```

Comment:
Minor: can we collapse the nested-ifs down to a single if statement now that we don't need this 'goto'?